whatlies/transformers/normalizer.py

```diff
@@ -0,0 +1,50 @@
+from sklearn.preprocessing import normalize
+
+from whatlies.transformers import Transformer
+from whatlies import EmbeddingSet
+from whatlies.transformers.common import embset_to_X, new_embedding_dict
+
+
+class Normalizer(Transformer):
+    """
+    This transformer normalizes the embeddings in an `EmbeddingSet` instance.
+
+    Arguments:
+        norm: the normalization value, could be either of `'l1'`, `'l2'` or `'max'`.
+        feature: if `True` each feature (i.e. dimension) of embeddings would be normalized
+            independently to unit norm; otheriwse, each embedding vector would be normazlied to unit norm.
+
+    Usage:
+
+    ```python
+    from whatlies.language import SpacyLanguage
+    from whatlies.transformers import Normalizer
+
+    words = ["prince", "princess", "nurse", "doctor", "banker", "man", "woman",
+             "cousin", "neice", "king", "queen", "dude", "guy", "gal", "fire",
+             "dog", "cat", "mouse", "red", "bluee", "green", "yellow", "water",
+             "person", "family", "brother", "sister"]
+
+    lang = SpacyLanguage("en_core_web_md")
+    emb = lang[words]
+
+    emb.transform(Normalizer(norm='l1'))
+    emb.transform(Normalizer(feature=True))
+    ```
+    """
+
+    def __init__(self, norm: str = "l2", feature: bool = False) -> None:
```

Comment:
I think the most common way (considering the visualisation use-case) of normalising here is to "make everything sum to 1". With that in mind, might it make sense to us `l1` as a norm?

Reply:
I am not sure, but if you say so I'll make the change.

Reply:
The reasoning is that cosine distances fit neatly in [0, 1] if the vectors sum up to one. That's how I realised that it deserves to be a feature.

Reply:
I made the change.
> The reasoning is that cosine distances fit neatly in [0, 1] if the vectors sum up to one.

Really?! Do you mean, in that case, the range of the range of cosine distance would be from 0 to 1 instead of -1 to 1?

Reply:
d0h. This was my bad. If two 2D vectors are positive they are [0, 1] but you're completely right. In general it should be [0,2]. 

I still prefer `l1` because it feels easier to reason about. It simply sums to one. I'll gladly be convinced if for word embeddings you feel `l2` is better.

Reply:
I see, that's right. Generally, I have mostly used `l2` norm; but I don't have a specific preference here.